docs/src/views/Layout.tsx

```diff
@@ -0,0 +1,212 @@
+  flex: link('Flex', '/components/flex'),
+  flexItem: link('Flex.Item', '/components/flex'),
+  grid: link('Grid', '/components/grid'),
+  segment: link('Segment', '/components/segment'),
+}
+
+export default () => (
+  <DocPage title="Layout">
+    <Header as="h2">Overview</Header>
+    <p>The following components are introduced by Stardust to handle layout aspects:</p>
+    <ul>
+      <li>{links.flex} - designed for layout in one dimension, either a row or a column.</li>
+      <li>
+        {links.grid} - designed for two-dimensional layout, rows and columns at the same time.
+      </li>
+    </ul>
+    <Header as="h3">Flex vs Grid</Header>
+    <p>
+      Sometimes it might not be evident if some particular case is a case of 1D (i.e. {code('Flex')}
+      ) or 2D (i.e. {code('Grid')}) layout. As a result, quite often {code('Grid')} component is
+      used at places where it is sufficient to use {code('Flex')}.
+    </p>
+    <p>Here is the question you might ask yourself to decide on that:</p>
+    <ul>
+      <li>
+        do I only need to control the layout by row <strong>or</strong> column? – if yes, use{' '}
+        {code('Flex')}.
+      </li>
+      <li>
+        do I need to control the layout by row <strong>and</strong> column? – if yes, use{' '}
+        {code('Grid')}.
+      </li>
+    </ul>
+    <Header as="h3">Note on Segment component</Header>
+    <p>
+      {links.segment} component may be abused for layout purposes, like in the following example:
+    </p>
+    <CodeSnippet
+      label="AbsolutePositioned.jsx"
+      value={`
+        import { Segment } from '@stardust-ui/react'
+
+        const OpaqueBackground = ({ children, color }) =>
+            <Segment styles={{ backgroundColor: color }} content={{ children }} />
+
+        const RelativePositioned = ({ children, top, right, bottom, left }) => (
+            <Segment styles={{ position: 'relative', top, right, bottom, left }} content={children} />
+        )
+      `}
+    />
+    <ExampleSnippet
+      render={() => {
+        const OpaqueBackground = ({ children, color }) => (
+          <Segment styles={{ backgroundColor: color }} content={{ children }} />
+        )
+
+        return (
+          <OpaqueBackground color="red">
+            This is a bad approach to opaque background :(
+          </OpaqueBackground>
+        )
+      }}
+    />
+    <ExampleSnippet
+      render={() => {
+        const RelativePositioned = ({ children, left }) => (
+          <Segment styles={{ position: 'relative', left }} content={children} />
+        )
+
+        return (
+          <RelativePositioned left={'30px'}>
+            This is a bad way to support relative-positioning :(
+          </RelativePositioned>
+        )
+      }}
+    />
+    <p>
+      While it might seem that the intent is addressed with the approach taken, however, this is
+      wrong for the following reason:
+    </p>
+    <blockquote>
+      Stardust makes it very important to{' '}
```

Comment:
```suggestion
      Stardust UI makes it very important to{' '}
```

_nit_

Reply:
it turns out that we are using Stardust/Stardust UI in our docs interchangeably. I would say that this is ok (due to that, when repeated frequently in text, "Stardust UI" is too formal and introduce unnecessary reading 'bumps') - but we might open a dedicated discussion on that. For now I would leave it as it is, given that this is an approach that is already taken - and will just follow it